src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -174,55 +169,53 @@ export class BuilderNavbarComponent implements OnDestroy {
     const errorPages = new Map<string, boolean>();
     const currentRoute = this.activatedRoute.snapshot.children[0].url[0].path;
 
-    this.store.submitForReview().then((canSubmit: boolean) => {
-      if (!canSubmit) {
-        // check for outcome errors
-        if (
-          this.validator.get('outcomes') ||
-          this.validator.outcomeValidator.errors.submitErrors.size
-        ) {
-          errorPages.set('outcomes', true);
-        }
-
-        // check for submission errors not related to outcomes
-        if (
-          this.validator.errors.submitErrors.size > 1 ||
-          (this.validator.errors.submitErrors.size === 1 &&
-            !this.validator.get('outcomes'))
-        ) {
-          errorPages.set('info', true);
-        }
+    if (!this.store.canSubmit()) {
```

Comment:
Most of the changes in this file are due to formatting. This is an actual change.